NWN.Permissions/src/ChatCommands/UserAddPermissionCommand.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+using Anvil.API;
+using Anvil.Services;
+using Jorteck.ChatTools;
+
+namespace Jorteck.Permissions
+{
+  [ServiceBinding(typeof(IChatCommand))]
+  internal class UserAddPermissionCommand : IChatCommand
+  {
+    private readonly ConfigService configService;
```

Comment:
This is partially my fault, as I already mixed these up across the codebase :P

Let's use the [Inject] property pattern for all of the command dependencies to be consistent, as commands will always run later anyway.

Reply:
78b475a0dc1a206a6bf1f96a1d8b550c322132b0